### PR TITLE
Update landing page

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- Updated Key features
- Updated In the nutshell
- Updated Manifesto

**Related issue(s)**
https://github.com/kyma-project/website/issues/679

### CODE REVIEW

File: content/i18n/en/landing-page.json [hunk 1]
```diff
@@ -27,32 +28,31 @@
...
+          "Kyma empowers developers to easy deploy workloads in form of functions. With Kyma you can focus on coding.",
+          "Simply write your function code, add a configuration manifest and push it to Kyma using supporting CLI or directly to git."
         ]
       },
       "runtime": {
-        "title": "Application runtime",
+        "title": "Eventing",
         "paragraphs": [
-          "Build standalone Kubernetes applications from scratch. Kyma is also an application runtime that is scalable, secure, and reliable. It allows you to develop microservices-based solutions in any language and run them on Kubernetes clusters from a cloud provider of your choice."
+          "Kyma allows you to trigger workloads ( functions or microservices ) via in-cluster events or extrenal business events"
```
Comment (tgorgol) [inline content/i18n/en/landing-page.json:42]:
```suggestion
          "Kyma allows you to trigger workloads (functions or microservices) via in-cluster events or external business events"
```
the space after parentheses causes some weird line breaks:
image

File: content/i18n/en/landing-page.json [hunk 2]
```diff
@@ -27,32 +28,31 @@
...
     "paragraph": "Kyma {spelling} is a platform for extending applications with serverless functions and microservices.",
-    "spelling": "/kee-ma/",
     "features": {
       "extensibility": {
-        "title": "Extensibility platform",
+        "title": "Serverless",
         "paragraphs": [
-          "If you are looking for cloud-native ways to extend your enterprise products, Kyma is a secure and tested environment you can rely on.",
-          "First, connect external APIs and ensure a stable event flow between the external or built-in services and your own applications. Then, start molding your own code into microservices and functions that extend your applications' logic."
+          "Kyma empowers developers to easy deploy workloads in form of functions. With Kyma you can focus on coding.",
```
Comment (tgorgol) [inline content/i18n/en/landing-page.json:35]:
```suggestion
          "Kyma empowers developers to easily deploy workloads in form of functions. With Kyma you can focus on coding.",
```

**Review by tgorgol — DISMISSED** [review]
LGTM 👍

File: content/i18n/en/landing-page.json [hunk 3]
```diff
@@ -1,8 +1,9 @@
 {
   "headline": "Extend your applications\nwith Kubernetes",
   "manifesto": {
-    "paragraph": "{brand} is a cloud-native application runtime that combines the power of Kubernetes with a set of best-in-class tools and open-source components, that empower you to develop, run and operate secure and scalable cloud-native applications.",
+    "paragraph": "{brand} {spelling} is a cloud-native application runtime that combines the power of Kubernetes with a set of best-in-class tools and open-source components, that empower you to develop, run and operate secure and scalable cloud-native applications.",
```
Comment (NHingerl) [inline content/i18n/en/landing-page.json:4]:
```suggestion
    "paragraph": "{brand} {spelling} is a cloud-native application runtime that combines the power of Kubernetes with a set of best-in-class tools and open-source components that empower you to develop, run and operate secure and scalable cloud-native applications.",
```

File: content/i18n/en/landing-page.json [hunk 4]
```diff
@@ -27,32 +28,31 @@
...
     "paragraph": "Kyma {spelling} is a platform for extending applications with serverless functions and microservices.",
-    "spelling": "/kee-ma/",
     "features": {
       "extensibility": {
-        "title": "Extensibility platform",
+        "title": "Serverless",
         "paragraphs": [
-          "If you are looking for cloud-native ways to extend your enterprise products, Kyma is a secure and tested environment you can rely on.",
-          "First, connect external APIs and ensure a stable event flow between the external or built-in services and your own applications. Then, start molding your own code into microservices and functions that extend your applications' logic."
+          "Kyma empowers developers to easily deploy workloads in form of functions. With Kyma you can focus on coding.",
```
Comment (NHingerl) [inline content/i18n/en/landing-page.json:35]:
```suggestion
          "Kyma empowers developers to easily deploy workloads in form of Functions. With Kyma, you can focus on coding.",
```

File: content/i18n/en/landing-page.json [hunk 5]
```diff
@@ -27,32 +28,31 @@
...
+          "Simply write your function code, add a configuration manifest and push it to Kyma using supporting CLI or directly to git."
```
Comment (NHingerl) [inline content/i18n/en/landing-page.json:36]:
```suggestion
          "Simply write your Function code, add a configuration manifest, and push it to Kyma using supporting CLI or directly to git."
```
For "supporting CLI", I think we need an article (_a_ or _the_), or if we refer to Kyma CLI specifically, write something like "using _our_ supporting CLI", or simply "using Kyma CLI".

File: content/i18n/en/landing-page.json [hunk 6]
```diff
@@ -27,32 +28,31 @@
...
         ]
       },
       "runtime": {
-        "title": "Application runtime",
+        "title": "Eventing",
         "paragraphs": [
-          "Build standalone Kubernetes applications from scratch. Kyma is also an application runtime that is scalable, secure, and reliable. It allows you to develop microservices-based solutions in any language and run them on Kubernetes clusters from a cloud provider of your choice."
+          "Kyma allows you to trigger workloads (functions or microservices) via in-cluster events or external business events"
```
Comment (NHingerl) [inline content/i18n/en/landing-page.json:42]:
```suggestion
          "With Kyma, you can trigger workloads (Functions or microservices) using in-cluster events or external business events."
```

(according [style guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/allow-allows) and [terminology](https://kyma-project.io/community/guidelines/content/#style-and-terminology-style-and-terminology-terminology)).

File: content/i18n/en/landing-page.json [hunk 7]
```diff
@@ -27,32 +28,31 @@
...
         ]
       },
       "support": {
-        "title": "Operational support",
+        "title": "Observability",
         "paragraphs": [
-          "Kyma ensures smooth Day-2 support for the standalone applications and extensions you'll create. Thanks to numerous CNCF open-source projects that come preconfigured with Kyma, you can easily monitor the performance of your microservices and functions, scan logs for errors, and upgrade your solutions painlessly."
+          "Kyma comes with pre-configured open-source observability tools. You can easily monitor the performance of your microservices and functions, scan logs for errors and trace request flow through entire system."
```
Comment (NHingerl) [inline content/i18n/en/landing-page.json:48]:
```suggestion
          "Kyma comes with pre-configured open-source observability tools. You can easily monitor the performance of your microservices and Functions, scan logs for errors, and trace the request flow through the entire system."
```

File: content/i18n/en/landing-page.json [hunk 8]
```diff
@@ -27,32 +28,31 @@
...
         ]
       }
     }
   },
   "inANutshell": {
-    "headline": "In a nutshell",
-    "paragraph": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+    "headline": "In the nutshell",
+    "paragraph": "With Kyma your workloads are in the center. Kyma helps you to expose their APIs, trigger them with events, enable them for external service consumption and will assure observability.",
```
Comment (NHingerl) [inline content/i18n/en/landing-page.json:55]:
```suggestion
    "paragraph": "With Kyma, your workloads are in the center. Kyma helps you to expose their APIs, trigger them with events, enable them for external service consumption and will assure observability.",
```

The sentence structure might confuse readers. It's basically "Kyma helps you to a, b, c, and [Kyma] will d." Would the last bit still be correct if we write simply "...enable ... consumption, _and assure observability_." ?

**Review by NHingerl — CHANGES_REQUESTED** [review]

File: content/i18n/en/landing-page.json (hunk 8 above)
Comment (kwiatekus) [reply content/i18n/en/landing-page.json]: yes. absolutely. I will push the changes

File: content/i18n/en/landing-page.json (hunk 5 above)
Comment (kwiatekus) [reply content/i18n/en/landing-page.json]: Added `our` to make it sound better. thx

**Review by tgorgol — APPROVED** [review]

File: content/i18n/en/landing-page.json [hunk 9]
```diff
@@ -27,32 +28,31 @@
...
-          "Kyma ensures smooth Day-2 support for the standalone applications and extensions you'll create. Thanks to numerous CNCF open-source projects that come preconfigured with Kyma, you can easily monitor the performance of your microservices and functions, scan logs for errors, and upgrade your solutions painlessly."
+          "Kyma comes with pre-configured open-source observability tools. You can easily monitor the performance of your microservices and Functions, scan logs for errors, and trace the request flow through the entire system."
         ]
       }
     }
   },
   "inANutshell": {
-    "headline": "In a nutshell",
-    "paragraph": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+    "headline": "In the nutshell",
```
Comment (alexandra-simeonova) [inline content/i18n/en/landing-page.json:54]:
```suggestion
    "headline": "In a nutshell",
```

"In a nutshell" is the correct expression 🙂  https://dictionary.cambridge.org/dictionary/english/in-a-nutshell

**Review by NHingerl — APPROVED** [review]

**Review by tgorgol — APPROVED** [review]

**Pull request merged**